lib/screens/history/transfer_history_screen.dart

```diff
@@ -160,30 +160,6 @@ class _TransferHistoryScreenState extends State<TransferHistoryScreen>
                 await historyProvider.getAllFileTransferHistory();
               },
               successBuilder: (provider) {
-                // if ((provider.displayFilesHistory.isEmpty)) {
```

Comment:
why is this commented ?

Reply:
This was from an old design 
There is no need to keep them here